From 83219d023b5b21826c7482d4aa3f6372f520825d Mon Sep 17 00:00:00 2001
From: Florian Festi <ffesti@redhat.com>
Date: Fri, 19 Feb 2016 09:56:31 +0100
Subject: [PATCH] Also block idle and sleep in the systemd-inhibit plugin

We really should not suspend or hibernate during rpm operations. Chances are
too high to not wake up properly and damage the system (see rhbz#1297984).
---
 plugins/systemd_inhibit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/systemd_inhibit.c b/plugins/systemd_inhibit.c
index b05d3d2..4a90c44 100644
--- a/plugins/systemd_inhibit.c
+++ b/plugins/systemd_inhibit.c
@@ -27,7 +27,7 @@ static int inhibit(void)
     }
 
     if (msg) {
-	const char *what = "shutdown";
+	const char *what = "idle:sleep:shutdown";
 	const char *mode = "block";
 	const char *who = "RPM";
 	const char *reason = "Transaction running";
-- 
2.5.0

